### ISSUE TITLE
Touch Effects: what to do about keyboard support for touch demo?

### ISSUE THREAD

**Issue by dstorey** [MicrosoftEdge/Demos-old#190]
https://developer.microsoft.com/en-us/microsoft-edge/testdrive/demos/toucheffects/

One can use mouse or touch to move the dots around. However, keyboard doesn't work. Not sure even how this should work with keyboard. Not sure if this should be exempt as it is a multi touch demo.

**Comment by molant** [comment]
@dstorey this is a touch demo. It doesn't make sense to use keyboard...

**Comment by dstorey** [comment]
@molant it is, but still needs to pass unless can get an exception. I think this is the easiest to justify. I think it is mostly that it is badly designed, in that if we want to teach touch for real developers, it should be an experience that falls back for other input like mouse any keyboard, so not teaching bad practices. In this case the only thing I can think of is treating the page like a grid were you can tab between different cells and make the dots move there as if it was touched. But that is probably fairly hard as would need a bunch of hidden buttons or whatever, and no idea what the screen reader would be for that.
